cocos/spine/assembler/simple.ts

```diff
@@ -169,14 +169,14 @@ function realTimeTraverse (comp: Skeleton): void {
     for (let i = 0; i < ic; i++) ibuf[i] += chunkOffset;
 
     const data = model.getData();
+    const textures = model.getTextures();
     const count = data.size();
     let indexOffset = 0;
     let indexCount = 0;
-    for (let i = 0; i < count; i += 6) {
+    for (let i = 0; i < count; i += 5) {
```

Comment:
Any reasons to change the step from 6 to 5?

Reply:
Currently, texture doesn`t store at data

Reply:
Got it.